Makefile

```diff
@@ -4,8 +4,14 @@ all:
 	go get ./...
 	go install ./...
 	cd extensions/sample; make
+	for CMD in `ls cmd` ; do (cd ./cmd/$$CMD; echo "install $$CMD"; go install ./...); done
 
 test:
 	# since some tests call separately-built binaries, clear the cache to ensure all get run
 	go clean -testcache
 	go test ./... -v
+	for CMD in `ls cmd` ; do (cd ./cmd/$$CMD; echo "test $$CMD"; go test ./... -v); done
```

Comment:
Is this necessary?
All tests are already run by
```go
go test ./... -v
```
right?

Reply:
@shenqidebaozi I have created a pull request for this:
https://github.com/shenqidebaozi/gnostic/pull/3

Reply:
@morphar - no, I don't think that's true. See [this blog post](https://medium.com/compass-true-north/catching-up-with-the-world-go-modules-in-a-monorepo-c3d1393d6024) for example (search for "go test").

Reply:
@timburks Sorry! You are right 👍 
I read the test output of `go test ./... -v` too quickly.

Reply:
@shenqidebaozi I have deleted the pull request.

Reply:
@morphar ok.